portal/src/components/Materials/CollectionList/CollectionList.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <h3 class="material__info_subttl">{{ $t('In-collection') }}</h3>
+    <ul class="material__info_collections">
+      <li
+        v-for="collection in collections"
+        :key="collection.id"
+        class="blue-bar material__info_collection"
+      >
+        <router-link
+          class="content"
+          :to="
+            localePath({
+              name: 'collections-id',
+              params: { id: collection.id }
+            })
+          "
+        >
+          <h3>
+            {{ collectionTitle(collection) }}
```

Comment:
Hoeft hier niet bij in welke community de collectie zit?

Reply:
in het ticket staat: "En daaronder moet de collectietitel weergegeven worden, in plaats van de communitynaam". Dus ik denk het niet